server/env.js

```diff
@@ -0,0 +1,32 @@
+// WARNING! All process.env values are strings, so check for 'true' or 'false'
+
+const {
+  NODE_ENV,
+  // OPEN_ALL_CONTESTS,
+  USE_FIREBASE,
+  ENV_LEVEL,
+  LOG_LEVEL: LOG_LEVEL_DOTENV,
+  ENV_PORT,
+} = process.env;
+
+const IS_DEV = NODE_ENV !== 'production';
+const TITLE = 'Vexillology Contests';
```

Comment:
This is no longer being read from `process.env`, but instead hard-coded to `'Vexillology Contests'` in all environments. Could do something like this instead:
```
const {
  ENV_LEVEL = 'prod',
  NODE_ENV,
  TITLE = 'Vexillology Contests',
} = process.env;

const IS_DEV = NODE_ENV !== 'production';

export default {
  ENV_LEVEL,
  IS_DEV,
  TITLE,
};
```

Reply:
Done